app/utils/chart-config/versioning.ts

```diff
@@ -2125,6 +2163,57 @@ export const configuratorStateMigrations: Migration[] = [
 
       delete newConfig.layout.blocks;
 
+      return newConfig;
+    },
+  },
+  {
+    description: `ALL {
+      interactiveFiltersConfig {
+        legend {
+          + visible
+        }
+      }
+    }`,
```

Comment:
@noahonyejese I am not sure why we have two migrations added here? When the feature has not landed on PROD yet, I think we should only have one migration to not complicate things 👍